--- Onesfto/Modules/GW2_UI/GW2_UI_1.15.9/core/GW2_ui.lua	2026-07-26 13:19:34
+++ GW2_UI/core/GW2_ui.lua	2026-07-26 08:02:59
@@ -682,7 +682,7 @@
 
     GW.LoadCharacter()
 
-    if GW.Retail or GW.TBC then
+    if GW.Retail or GW.TBC or GW.Classic then
         GW.LoadSocialFrame()
     end
 
@@ -907,14 +907,15 @@
         GW.LoadCalendarSkin()
     else
         GW.LoadQuestLogFrameSkin()
+        if GW.LoadTradeSkillFrameSkin then GW.LoadTradeSkillFrameSkin() end
         GW.LoadQuestTimersSkin()
     end
 
     if not (GW.Classic or GW.TBC) then
         GW.MakeAltPowerBarMovable()
         GW.LoadLFGSkins()
-        GW.LoadMailSkin()
     end
+    GW.LoadMailSkin()
 
     if not (GW.Classic or GW.TBC or GW.Wrath) then
         GW.LoadSocketUISkin()
