--- Onesfto/Modules/GW2_UI/GW2_UI_1.15.9/Games/Shared/Immersive/Questing/ImmersiveQuestingFrame.lua	2026-07-26 13:19:34
+++ GW2_UI/Games/Shared/Immersive/Questing/ImmersiveQuestingFrame.lua	2026-07-26 10:45:28
@@ -311,6 +311,9 @@
     if _G.GwImmersiveQuestDebugFrame then
         _G.GwImmersiveQuestDebugFrame:Hide()
     end
+    if not InCombatLockdown() then
+        self:SetPropagateKeyboardInput(true)
+    end
     self:EnableKeyboard(false)
     self:SetScript("OnKeyDown", nil)
     if self.was_showing then
