--- Onesfto/Modules/GW2_UI/GW2_UI_1.15.9/settings/panels/panel_auras.lua	2026-07-26 13:19:34
+++ GW2_UI/settings/panels/panel_auras.lua	2026-07-26 14:19:30
@@ -14,7 +14,7 @@
     p_auras.sub:SetFont(UNIT_NAME_FONT, 12)
     p_auras.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     p_auras.sub:SetText(L["Edit general unitframe aura settings."])
-    p_auras.header:SetWidth(p_auras.header:GetStringWidth())
+    p_auras.header:SetWidth(p_auras.header:GetStringWidth() + 10)
     p_auras.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     p_auras.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     p_auras.breadcrumb:SetText(GENERAL)
@@ -27,7 +27,7 @@
     p_indicator.sub:SetFont(UNIT_NAME_FONT, 12)
     p_indicator.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     p_indicator.sub:SetText(L["Edit raid aura indicators."])
-    p_indicator.header:SetWidth(p_indicator.header:GetStringWidth())
+    p_indicator.header:SetWidth(p_indicator.header:GetStringWidth() + 10)
     p_indicator.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     p_indicator.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     p_indicator.breadcrumb:SetText(L["Raid Indicators"])
@@ -40,7 +40,7 @@
     p_missingBuffs.sub:SetFont(UNIT_NAME_FONT, 12)
     p_missingBuffs.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     p_missingBuffs.sub:SetText(L["Edit raid buff bar."])
-    p_missingBuffs.header:SetWidth(p_missingBuffs.header:GetStringWidth())
+    p_missingBuffs.header:SetWidth(p_missingBuffs.header:GetStringWidth() + 10)
     p_missingBuffs.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     p_missingBuffs.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     p_missingBuffs.breadcrumb:SetText(L["Missing Raid Buffs"])
