--- Onesfto/Modules/GW2_UI/GW2_UI_1.15.9/settings/panels/panel_interface_features.lua	2026-07-26 13:19:34
+++ GW2_UI/settings/panels/panel_interface_features.lua	2026-07-26 07:37:55
@@ -18,7 +18,7 @@
     p:AddOption(TALENTS, L["Enable the talents, specialization, and spellbook replacement."], {getterSetter = "USE_TALENT_WINDOW", callback = function() GW.ShowRlPopup = true end, groupHeaderName = L["UI Windows"], hidden = GW.Retail, isMasterToggle = true})
     p:AddOption(SPELLBOOK_ABILITIES_BUTTON, nil, {getterSetter = "USE_SPELLBOOK_WINDOW", callback = function() GW.ShowRlPopup = true end, groupHeaderName = L["UI Windows"], hidden = GW.Retail, isMasterToggle = true})
     p:AddOption(TRADE_SKILLS, L["Enable the profession replacement."], {getterSetter = "USE_PROFESSION_WINDOW", callback = function() GW.ShowRlPopup = true end, groupHeaderName = L["UI Windows"], hidden = not GW.Retail, isMasterToggle = true})
-    p:AddOption(FRIENDS, nil, {getterSetter = "USE_SOCIAL_WINDOW", callback = function() GW.ShowRlPopup = true end, groupHeaderName = L["UI Windows"], hidden = not (GW.Retail or GW.TBC), isMasterToggle = true})
+    p:AddOption(FRIENDS, nil, {getterSetter = "USE_SOCIAL_WINDOW", callback = function() GW.ShowRlPopup = true end, groupHeaderName = L["UI Windows"], hidden = not (GW.Retail or GW.TBC or GW.Classic), isMasterToggle = true})
 
     p:AddGroupHeader(L["Components"])
     p:AddOption(INVENTORY_TOOLTIP, L["Enable the unified inventory interface."], {getterSetter = "BAGS_ENABLED", callback = function() GW.ShowRlPopup = true end, groupHeaderName = L["Components"], incompatibleAddons = "Inventory", isMasterToggle = true})
