--- Onesfto/Modules/GW2_UI/GW2_UI_1.15.9/settings/panels/panel_skins.lua	2026-07-26 13:19:34
+++ GW2_UI/settings/panels/panel_skins.lua	2026-07-26 14:19:30
@@ -11,7 +11,7 @@
     panel.sub:SetFont(UNIT_NAME_FONT, 12)
     panel.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     panel.sub:SetText(L["Adjust Skin settings."])
-    panel.header:SetWidth(panel.header:GetStringWidth())
+    panel.header:SetWidth(panel.header:GetStringWidth() + 10)
     panel.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     panel.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     panel.breadcrumb:SetText(breadcrumb)
@@ -36,11 +36,12 @@
     general:AddOption(L["Talking Head"], nil, {getterSetter = "TALKINGHEAD_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end, hidden = not GW.Retail})
     general:AddOption(L["Misc Frames"], nil, {getterSetter = "MISC_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end})
     general:AddOption(ADDON_LIST, nil, {getterSetter = "ADDONLIST_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end})
-    general:AddOption(MINIMAP_TRACKING_MAILBOX, nil, {getterSetter = "MAIL_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end, hidden = GW.Classic or GW.TBC or GW.Wrath})
     general:AddOption(HELP_FRAME_TITLE, nil, {getterSetter = "HELPFRAME_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end})
     general:AddOption(TIMEMANAGER_TITLE, nil, {getterSetter = "TIMEMANAGER_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end})
 
     gameFrames:AddOption(QUEST_LOG, nil, {getterSetter = "QUESTLOG_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end, hidden = GW.Retail})
+    gameFrames:AddOption(TRADE_SKILLS, nil, {getterSetter = "TRADESKILL_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end, hidden = GW.Retail})
+    gameFrames:AddOption(MINIMAP_TRACKING_MAILBOX, nil, {getterSetter = "MAIL_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end})
     gameFrames:AddOption(FLIGHT_MAP, nil, {getterSetter = "FLIGHTMAP_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end})
     gameFrames:AddOption(MACRO, nil, {getterSetter = "MACRO_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end})
     gameFrames:AddOption(BARBERSHOP, nil, {getterSetter = "BARBERSHOP_SKIN_ENABLED", callback = function() GW.ShowRlPopup = true end, hidden = not GW.Retail})
