--- Onesfto/Modules/GW2_UI/GW2_UI_1.15.9/settings/panels/panel_raid.lua	2026-07-26 13:19:34
+++ GW2_UI/settings/panels/panel_raid.lua	2026-07-26 14:19:30
@@ -74,7 +74,7 @@
     general.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     general.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    general.header:SetWidth(general.header:GetStringWidth())
+    general.header:SetWidth(general.header:GetStringWidth() + 10)
     general.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     general.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     general.breadcrumb:SetText(GENERAL)
@@ -131,7 +131,7 @@
     raid10.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     raid10.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    raid10.header:SetWidth(raid10.header:GetStringWidth())
+    raid10.header:SetWidth(raid10.header:GetStringWidth() + 10)
     raid10.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     raid10.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     raid10.breadcrumb:SetText(RAID..": 10")
@@ -229,7 +229,7 @@
     raid25.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     raid25.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    raid25.header:SetWidth(raid25.header:GetStringWidth())
+    raid25.header:SetWidth(raid25.header:GetStringWidth() + 10)
     raid25.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     raid25.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     raid25.breadcrumb:SetText(RAID..": 25")
@@ -327,7 +327,7 @@
     raid40.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     raid40.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    raid40.header:SetWidth(raid40.header:GetStringWidth())
+    raid40.header:SetWidth(raid40.header:GetStringWidth() + 10)
     raid40.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     raid40.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     raid40.breadcrumb:SetText(RAID..": 40")
@@ -421,7 +421,7 @@
     tank.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     tank.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    tank.header:SetWidth(tank.header:GetStringWidth())
+    tank.header:SetWidth(tank.header:GetStringWidth() + 10)
     tank.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     tank.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     tank.breadcrumb:SetText(MAINTANK)
@@ -514,7 +514,7 @@
     p.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     p.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    p.header:SetWidth(p.header:GetStringWidth())
+    p.header:SetWidth(p.header:GetStringWidth() + 10)
     p.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     p.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     p.breadcrumb:SetText(PET)
@@ -599,7 +599,7 @@
     party.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     party.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    party.header:SetWidth(party.header:GetStringWidth())
+    party.header:SetWidth(party.header:GetStringWidth() + 10)
     party.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     party.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     party.breadcrumb:SetText(PARTY)
@@ -700,7 +700,7 @@
     p.sub:SetTextColor(181 / 255, 160 / 255, 128 / 255)
     p.sub:SetText(L["Edit the party and raid options to suit your needs."])
 
-    p.header:SetWidth(p.header:GetStringWidth())
+    p.header:SetWidth(p.header:GetStringWidth() + 10)
     p.breadcrumb:SetFont(DAMAGE_TEXT_FONT, 12)
     p.breadcrumb:SetTextColor(GW.Colors.TextColors.LightHeader:GetRGB())
     p.breadcrumb:SetText(PARTY .. " " ..PET)
